source/tests/consistent/descriptor/test_se_e2_a.py

```diff
@@ -147,3 +152,35 @@ def eval_pt(self, pt_obj: Any) -> Any:
 
     def extract_ret(self, ret: Any, backend) -> Tuple[np.ndarray, ...]:
         return (ret[0],)
+
+    @property
+    def rtol(self) -> float:
+        """Relative tolerance for comparing the return value."""
+        (
+            resnet_dt,
+            type_one_side,
+            excluded_types,
+            precision,
+        ) = self.param
+        if precision == "float64":
+            return 1e-10
+        elif precision == "float32":
+            return 1e-4
+        else:
+            raise ValueError(f"Unknown precision: {precision}")
+
+    @property
+    def atol(self) -> float:
+        """Absolute tolerance for comparing the return value."""
+        (
+            resnet_dt,
+            type_one_side,
+            excluded_types,
+            precision,
+        ) = self.param
+        if precision == "float64":
+            return 1e-10
+        elif precision == "float32":
+            return 1e-4
+        else:
+            raise ValueError(f"Unknown precision: {precision}")
```

Comment:
These two properties not used yet?

Reply:
It's used here: https://github.com/deepmodeling/deepmd-kit/blob/604679c8a38eaa2403a9233c8e4d3a6c54792617/source/tests/consistent/common.py#L257